resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -99,6 +99,14 @@ private[spark] object Config extends Logging {
       .toSequence
       .createWithDefault(Nil)
 
+  val CONFIG_MAP_MAXSIZE =
+    ConfigBuilder("spark.kubernetes.configMap.maxSize")
+      .doc("Max size limit for a config map. This is configurable as per" +
+        " https://etcd.io/docs/v3.4.0/dev-guide/limit/ on k8s server end.")
+      .version("3.1.0")
+      .longConf
+      .createWithDefault(1573000) // 1.5 MiB
```

Comment:
Is this correct? The spec means 1.5Mib = `1572864`. This value seems to be greater than that and will not prevent the failure.

Reply:
@dongjoon-hyun I think you are correct about this (though it is 1.5MiB not 1.5Mib Mebibyte v/s Mebibit). I was deceived by google:
![Untitled](https://user-images.githubusercontent.com/992952/103396327-91305080-4b58-11eb-8392-5276701451dd.png)
Correct value is : 1.5MiB= 1572864 bytes